src/labels/CustomLabels.labels

```diff
@@ -300,6 +300,14 @@
         <shortDescription>AssistiveTextError</shortDescription>
         <value>Expired</value>
     </labels>
+    <labels>
```

Comment:
@force2b , @sam-knox : even though this PR is work in progress, can you please review three new custom labels so @meena-sfdc  can add them into the label freeze PR?